bot/cogs/free.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from datetime import datetime
+
+from discord import Colour, Embed, Member, utils
+from discord.ext.commands import BucketType, Context, command, cooldown
+
+from bot.constants import Categories, Free
+
+
+log = logging.getLogger(__name__)
+
+PYTHON_HELP_ID = Categories.python_help
+TIMEOUT = Free.activity_timeout
+
+
+class Free:
+    """Tries to figure out which help channels are free."""
+    @command(name="free", aliases=('f',))
```

Comment:
I think when there's only a single alias, having it be a tuple is pointless. the readability is worse due to the trailing comma, and as we've explored in previous PRs, there's no real difference in performance anyway.

Reply:
okay, I'll buy it.